corehq/apps/app_manager/static/app_manager/js/import_app.js

```diff
@@ -1,25 +1,8 @@
 hqDefine('app_manager/js/import_app', function () {
-    function CompressionViewModel(source, post_url) {
-        var self = this;
-        self.name = ko.observable("");
-        self.source = ko.observable(source);
-        self.save = function (formElement) {
-            $.postGo(post_url, {
-                name: self.name(),
-                compressed: hqImport("hqwebapp/js/lib/compression").LZW.compress(self.source()),
```

Comment:
This is the only place `compression.js` is used, so you can remove the file itself and its script tag.

It's been on the js backlog for a while to replace `compression.js` with a standard library (it's currently a copy-paste of third party code) so it's nice to see it instead just disappear.

Reply:
Sure!